code/modules/paperwork/folders.dm

```diff
@@ -151,4 +151,79 @@
 
 /obj/item/folder/embedded/handle_post_remove()
 	if(!length(contents))
-		qdel(src)
\ No newline at end of file
+		qdel(src)
+
+/obj/item/folder/envelope
+	name = "envelope"
+	desc = "A thick envelope. You can't see what's inside."
+	icon_state = "envelope_sealed"
+	var/sealed = 1
+
+/obj/item/folder/envelope/update_icon()
+	if(sealed)
+		icon_state = "envelope_sealed"
+	else
+		icon_state = "envelope[contents.len > 0]"
+
+/obj/item/folder/envelope/examine(mob/user)
+	. = ..()
+	to_chat(user, "The seal is [sealed ? "intact" : "broken"].")
+
+/obj/item/folder/envelope/proc/sealcheck(user)
+	var/ripperoni = alert("Are you sure you want to break the seal on \the [src]?", "Confirmation","Yes", "No")
+	if(ripperoni == "Yes")
+		visible_message(SPAN_NOTICE("[user] breaks the seal on \the [src], and opens it."))
+		sealed = FALSE
+		update_icon()
+		return TRUE
+
+/obj/item/folder/envelope/attack_self(mob/user)
+	if(sealed)
+		sealcheck(user)
+		return
+	else
+		..()
+
+/obj/item/folder/envelope/attackby(obj/item/W, mob/user)
+	if(sealed)
+		sealcheck(user)
+		return
+	else
+		..()
+
+/obj/item/folder/envelope/zta
+	name = "leviathan zero-point artillery instructions"
+	desc = "A small envelope. This one reads 'SCC - CONFIDENTIAL'."
+
+/obj/item/folder/envelope/zta/Initialize()
+	. = ..()
+	var/obj/item/paper/R = new(src)
+	R.set_content("leviathan zero-point artillery instructions", "<table><cell><hr><small><center><img src=scclogo.png><br><b>Stellar Corporate Conglomerate<br>\
+	SCCV Horizon</b><hr><b>Form 0000<br>\
+	<large>Confidential Information Report</large></b></center><hr>\
+	<b>Classification Index:</b> <font color='red'>TOP SECRET</font>, protect at all costs <field><hr><b>Entrusted Personnel:</b> Command, Engineering<br> <field>\
```

Comment:
```suggestion
	<b>Classification Index:</b> <font color='red'>TOP SECRET</font>, protect at all costs <field><hr><b>Entrusted Personnel:</b> Command<br> <field>\	
```
I understand the reasoning (engineers need to know how it works to know how to fix), but giving even the engineering apprentice the notion that they can read this 'top secret' document I feel isn't the best idea.

I recommend just having it so they will learn about that part of the system if they need to know. Best to just say command is entrusted with the document itself, and they can give information to engineering about it if neccessary.